search/update_attribute_configuration.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.api_core.client_options import ClientOptions
+from google.cloud.retail_v2 import ProductServiceClient, Product, UpdateProductRequest, GetProductRequest, \
+    CustomAttribute
+
+# TODO Define the project number here:
+project_number = " "
+endpoint = "retail.googleapis.com:443"
```

Comment:
in some samples you pass this directly into ClientOptions, in some you have variable like here. Also in some samples you specify port, in some not.
Let's unify this